### ISSUE TITLE
About the details of learning rate

### ISSUE THREAD

**Issue by hongxin001** [xjtushujun/meta-weight-net#7]
There is a sentence in the appendix:  "With batch normalization, we effectively cancel the learning rate of Meta-Weight-Net, and it works well with a fixed learning rate. "

I'm not sure what it is about. Would you please give an explanation in detail?  Does it mean we don't need to fine-tune the learning rate of meta networks because of BN?

**Comment by shanshuo** [comment]
@xjtushujun Thanks for sharing the code of your nice work. For this part, I also have questions as @hongxin001 . 
1. Why batch normalization could cancel the learning rate of MV-Net? 
2. In the old version, you normalize the weight by its sum: https://github.com/xjtushujun/meta-weight-net/blob/bd1fd3e297c59df7c5264fb636c67d8ee03bcf0d/train_WRN-28-10_Meta_PGC.py#L175 While in the stable version of MV-Net you didn't: https://github.com/xjtushujun/meta-weight-net/blob/3f1800ff26bc66ceda6075f3fa25f3a7e0665a4d/MW-Net.py#L207 Why new version is more stable? How does it avoid the output weight becoming all zeros?
